src/Components/RoundOverview.js

```diff
@@ -29,11 +29,11 @@ const RoundOverview = () => {
   return (
     <div>
       <h2>{location.state.course_name}</h2>
-      <p>Par: {location.state.total_par}</p>
-      <p>Date: {location.state.date}</p>
+      <p className='round-par'>Par: {location.state.total_par}</p>
```

Comment:
Thank you for adding className on our elements that need to be styled for this component